src/main/resources/openapi/token-1.0.yaml

```diff
@@ -0,0 +1,110 @@
+openapi: 3.0.0
```

Comment:
Is this ready to add a Jenkinsfile entry that lints and published the openapi files?
https://dev.folio.org/guides/jenkinsfile/#back-end-modules

Reply:
Yes I think so. Will modify the Jenkins file.

Reply:
Added. After moving the directories around the linting seems to be working. See what you think.

Reply:
Yes. I viewed the Artifacts apiLintReport. All good.